### PR TITLE
users organizations and groups in Admin section 

### LINKED ISSUE #273: Admin section-users-organizations+groups VS Home section organization+groups
Admin section-users-organizations+groups

- these two components show VOs and groups of the user that is logged in not the user that admin wants to show. That is bug. These two components should show Vos and groups of the previewed user. 
- create new components for this or edit components that are used now - depeds what will be easier to read or easier to create

Home section organization+groups

- rename items in side-menu in home section to My organizations, My groups, My settings
- also if possible change icon of Home section to something like this https://material.io/resources/icons/?icon=home&style=baseline

### PR DESCRIPTION
**Pull request by DanoFecko**
resolves #273 
* fixed bug where users groups and VOs are not the ones from shown user but from principal
* when using user-organizations and user-groups components you have to add parameter to routing module (see  changed code), that determines if you want to show entities related to specified user or principal.
* changed home icon in side menu
* changed subTitles in Side-menu/Home (organizations => my organizations, ...)

### CODE REVIEW

**Comment by DanoFecko** [comment]
@xkureck It's fixed

File: apps/admin-gui/src/app/shared/components/user-detail-page/user-organizations/user-organizations.component.html [hunk 1]
```diff
@@ -1,11 +1,12 @@
-<h1 class="page-subtitle">{{'SHARED_LIB.PERUN.ORGANIZATIONS.TITLE' | translate}}</h1>
+<!--<h1 class="page-subtitle">{{'SHARED_LIB.PERUN.ORGANIZATIONS.TITLE' | translate}}</h1>-->
```
Comment (xkureck) [inline apps/admin-gui/src/app/shared/components/user-detail-page/user-organizations/user-organizations.component.html:1]: remove this row if its not needed

**Comment by DanoFecko** [comment]
@xkureck I fixed the issues, but don't do the review yet. I also have to do the repairs to user facility attributes.

**Comment by DanoFecko** [comment]
I also fixed user facility attributes

**Pull request merged**